neqo-client/src/main.rs

```diff
@@ -7,6 +7,7 @@
 #![deny(warnings)]
 use neqo_common::Datagram;
 use neqo_crypto::init_db;
+use neqo_http3::request_stream_server::Header;
```

Comment:
re-export `Header` at top of http3 so apps can just do `use neqo_http3::Header`?